### ISSUE TITLE
Error in ARAX query from Workflow C.1

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX#1625]
https://github.com/NCATSTranslator/minihackathons/issues/223

**Comment by amykglen** [comment]
wondering if @chunyuma / @dkoslicki have had a chance to look into this? it came up again at today's Translator minihackathon

**Comment by chunyuma** [comment]
Hi @amykglen, based on my investigation, the reason that we have no result while COHD API has results might be due to no OMOP id mapping like #1571, #1574 and #1579. Note that it might has multiple reasons causing this issue (e.g. we removed all 'SNOMED CT' ids, our OMOP id mapping is not exactly the same as what COHD API uses). Can we have a way to know what COHD API returns for this query? I checked their APIs and it seems like there is no API that can take the query graph. 

Another solution that might solve this problem is to directly call their APIs to get the OMOP mapping ids that they use.
